### PR TITLE
Add sequelize & migration

### PR DESCRIPTION
**Pull request by inDream**
Connects likecoin/puttyimages-web#5

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  development: {
```
Comment (rickmak) [inline config/database.js:2]: I think `docker` is a good env name in current setup. It tells this setting is assuming docker setup and the setting are passing via envvar.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+      - ./.nuxt:/usr/src/app/.nuxt
+      - ./assets:/usr/src/app/assets
+      - ./build:/usr/src/app/build
+      - ./components:/usr/src/app/components
+      - ./layouts:/usr/src/app/layouts
+      - ./migrations:/usr/src/app/migrations
+      - ./pages:/usr/src/app/pages
+      - ./plugins:/usr/src/app/plugins
+      - ./server:/usr/src/app/server
+      - ./static:/usr/src/app/static
```
Comment (rickmak) [inline docker-compose.yml]: Why we want this?

I assume you want to keep the node_modules inside docker and outside docker is separated.

This setup have potential when developer adding a new folder and forgot to add here. I prefer the original setup.

File: migrations/20180601081626-create-user.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      name: {
+        allowNull: false,
+        type: Sequelize.TEXT,
+      },
+    })
+      .then(() => queryInterface.bulkInsert('user', [
+        { name: 'Alexandre' },
+        { name: 'Pooya' },
+        { name: 'Sébastien' },
+      ])),
```
Comment (rickmak) [inline migrations/20180601081626-create-user.js]: Seed data and migration is better separated.

File: docker-compose.yml [hunk 4]
```diff
@@ -1,16 +1,25 @@
-version: '2'
+version: '3'
```
Comment (rickmak) [inline docker-compose.yml:1]: Good, Sorry I don't use the latest version.

**Review by rickmak — CHANGES_REQUESTED** [review]
Please try to separate the seed data and migration.

File: config/database.js (hunk 1 above)
Comment (inDream) [reply config/database.js:2]: sequelize-cli use `development` as config key, need to change `NODE_ENV` or add env argument if want to change this.

File: docker-compose.yml (hunk 2 above)
Comment (inDream) [reply docker-compose.yml]: Map node_modules into docker is slow

**Pull request merged**